WalletWasabi/Crypto/Randomness/MockRandom.cs

```diff
@@ -1,19 +1,27 @@
+using NBitcoin.Secp256k1;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace WalletWasabi.Crypto.Randomness
 {
 	public class MockRandom : IWasabiRandom
 	{
+		public List<byte[]> GetBytesResults { get; } = new List<byte[]>();
+
 		public void GetBytes(byte[] output)
 		{
-			throw new NotImplementedException();
+			var first = GetBytesResults.First();
```

Comment:
We could get an EmptySequenceException here if the GetBytesResults is empty.

Reply:
It's a mock, so I don't care.